.github/workflows/main.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python_version: ['3.8', '3.9', '3.10', '3.11']
+        python_version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Is there a reason, you removed 3.8 from the tests?

Reply:
Yeah I should put that back, with some repos careful to include minimal set due to time, but this is pretty quick.